src/components/Widget/index.tsx

```diff
@@ -36,7 +36,7 @@ import { useSyncWidgetSettings } from './settings'
 import { DARK_THEME, LIGHT_THEME } from './theme'
 import { useSyncWidgetTransactions } from './transactions'
 
-export const DEFAULT_WIDGET_WIDTH = 360
+const DEFAULT_WIDGET_WIDTH = 360
```

Comment:
nit: it would be great to keep this in sync with `TokenDetails/Skeleton`, so they never run out of sync. 

In reality, this is just bad design, because it seems to me that we have two different components, TokenDetails "main page" and TokenDetails "skeleton".

I believe, instead, we should have a single component, TokenDetails, where we would have "RightPanel" (with fixed width) and inside, based on the loading state, we would either render WidgetSkeleton or Widget. That way, both WidgetSkeleton and Widget would not have to set width, but use 100% of available space and delegate it back to RightPanel to decide. 

I don't know whether this change is possible within this PR, probably not, but I would leave a TODO comment + Jira ticket to refactor this, because I think this can be improved.

Reply:
agreed i was also really confused by it when i was refactoring! will make a linear ticket

Reply:
I usually add it to the code too, but don't want you to re-request another review from Connor so just for the future haha